frontend/lib/common-steps/landlord-name-and-contact-types.tsx

```diff
@@ -44,9 +44,11 @@ const ReadOnlyLandlordDetails: React.FC<
         landlord={landlord}
       />
       <p>
-        If you feel strongly that this information is incorrect or incomplete,
-        however, you can{" "}
-        <Link to={forceManualHref}>provide your own details</Link>.
+        <Trans>
+          If you feel strongly that this information is incorrect or incomplete,
+          however, you can{" "}
+          <Link to={forceManualHref}>provide your own details</Link>.
+        </Trans>
```

Comment:
NOICE.

Um... I only just now realized that this isn't in Spanish for norent CA users!!  oof.  So thanks for i18n'ing this.